docs/Simulator.md

```diff
@@ -0,0 +1,30 @@
+# Simulator 
+
+The monoDrive Simulator provides users with the ability to test AV algorithms in one of several pre-made monoDrive Simulator levels. The Open Source monoDrive clients provide an API and examples for connecting to the Simulator to simulate traffic scenarios and replaying monoDrive trajectory files. The clients allow users to configure and test any number of configurations of monoDrive high-fidelity sensors.
+
+## Getting Started
+
+1. Install Unreal Engine **4.23** from [here](https://www.unrealengine.com/en-US/).
+
+    - In Epic Games Launcher, and select Marketplace >> Browse. Search for "VehicleSim Dynamics" and download this free plugin. 
```

Comment:
Probably should just be an additional step called "Install the CarSim Vehicle Dynamics plugin..."